src/twister.h

```diff
@@ -8,8 +8,12 @@
 
 #define LIBTORRENT_PORT_OFFSET 1000
 
-#define USERPOST_FLAG_RT 0x01
-#define USERPOST_FLAG_DM 0x02
+#define USERPOST_FLAG_RT    0x01
+#define USERPOST_FLAG_DM    0x02
+#define USERPOST_FLAG_FAV   0x04
+#define USERPOST_FLAG_P_FAV 0x0C
```

Comment:
yes, that was for fetching together the **fav** and the **pfav** messages with a single *get_pieces* call in *getfavs* method.